### ISSUE TITLE
[nll] Unweighted RMS computation

### ISSUE THREAD

**Issue by luca-s** [swiss-seismological-service/sed-SeisComP-contributions#12]
Following the discussion in [here](https://github.com/alomax/NonLinLoc/issues/29) we should consider if we want to compute the RMS in NLL plugin without weighting the residuals by arrival weight.

**Comment by luca-s** [comment]
Since changing the RMS computation might be consider a backward inconsistency we could consider to apply this change in the next release of SeisComP when the new NonLinLoc v7 code will land. That is already a big change

**Comment by gempa-jabe** [comment]
Will NonLinLoc 7 break catalogues by producing different solutions or different error measures? I am fine with changing the way the RMS is being computed, I just don't know how much it will affect current users.

Actually the name `standardError` suggests that the determination is kind of standard.

**Comment by luca-s** [comment]
> Will NonLinLoc 7 break catalogues by producing different solutions or different error measures? 

In my comparison tests between current NLL and NLL v7 I got similar results so I expect a transition to NonLinLoc 7 without any  backward incompatibilities: neither in behaviour nor in configuration. NLL v7 should be a drop-in replacement to the current version. However I cannot be sure 100%  since my testing is limited to my use cases.

>  I am fine with changing the way the RMS is being computed, I just don't know how much it will affect current users.

That would affect SED for example. So I pointed my collegues to this conversation. However for the sake of consistency I believe we should recompute RMS in a standard way.
